docs/source/getting-started.rst

```diff
@@ -19,10 +19,10 @@ Installation
 _______________________________________________
 
 
-Api Key
+Api Key / Access Token
```

Comment:
Authorization Token

Reply:
So is the header there just "Authorization Token" or "Api Key / Authorization Token"